backend/api.py

```diff
@@ -251,3 +252,21 @@ def frontend_review() -> FeedbackSchema:
     with open(random_feedback_file) as random_file:
         content = json.loads(random_file.read())
         return FeedbackSchema(**content)
+
+
+@app.get("/frontend/download", response_class=StreamingResponse)
```

Comment:
I think this shouldn't be `/frontend/download` but simply just `/download`. My intentions for the `/frontend` routes was to be routes that are called from the Javascript code and returning JSON, so that Javascript can render the response without page refresh.

In this case, we will have a simple `<a href="/download">Yo, download the data</a>`

Reply:
Oh, I didn't know it's so ez. I thought I have to call it in clojure :D